www/nginx/src/opnsense/service/templates/OPNsense/Nginx/location.conf

```diff
@@ -178,9 +178,9 @@ location {{ location.matchtype }} {{ location.urlpattern }} {
     proxy_ssl_server_name on;
 {%     else %}
     proxy_ssl_server_name off;
+{%    endif %}
 {%         if upstream.tls_name_override is defined and upstream.tls_name_override != '' %}
     proxy_ssl_name {{ upstream.tls_name_override }};
-{%         endif %}
 {%     endif%}
```

Comment:
The `endif` in line 184 should be indented by one level, right? I think it belongs to line 182.

Reply:
Hi! now line 184 closes "if->else" statement started at 177.
In my variant line 181 closes If->else statement started at line 177. so proxy_ssl_name can be applied regardless of proxy_ssl_server_name state (for now proxy_ssl_name applied only if proxy_ssl_server_name "off". so doesn't actually work). 
line 184 closes 182 "if"

Reply:
yes, sorry, I should have indent  the `endif` on 184 line

Reply:
@8191 
sorry, I have already lost a branch for this PR and cannot edit this. can you add this changes to your branch or should i make new PR?

Reply:
@fabianfrz Your review is still missing here, therefore a specific question:
Initially (before this PR) the `proxy_ssl_name` was only set if `proxy_ssl_server_name` was not set. Was this intentional, or an error?
I'm asking because due to the indentation it might also well be intentionally and the error could well simply be a missing "endif" for the proxy_ssl_server_name block...

For me the proposed correction of @kulikov-a makes sense.

Reply:
@8191 My intention was the following:
```
| SNI override enabled | disabled                 | HTTP host override enabled | disabled                     |
| SNI configured       | the one from the request | configured one             | the one from the host header |
```